URA-695/cell_line_analysis.py

```
@@ -0,0 +1,258 @@
+    # Create a figure and axis
+    fig, axes = plt.subplots(nrows=1, ncols=3, figsize=(15, 5))
+
+    for ax, group_data, title in zip(axes, data, [query_col, query_col2, truth_col]):
+        sns.boxplot(data=group_data, ax=ax, color='lightblue')
+        sns.stripplot(data=group_data, ax=ax, color='black', alpha=0.5, jitter=True)
+        ax.set_title(title)
+        ax.set_ylabel('Allele Frequency')
+        ax.set_xticks([]) 
+        ax.text(0.1, 0.95, 'n: {}'.format(len(group_data)),
+            horizontalalignment='left', verticalalignment='top',
+            transform=ax.transAxes, fontsize=12, bbox=dict(facecolor='white', alpha=0.5))
```

Ruff (0.8.0)

Unnecessary `dict` call (rewrite as a literal)

Rewrite as a literal

(C408)